### ISSUE TITLE
TypeError: result.value.split is not a function

### ISSUE THREAD

**Issue by eh1160** [kucherenko/jscpd#410]
I get the following error: 
```
TypeError: result.value.split is not a function
    at calculateLocation (C:\Users\ehammes\AppData\Roaming\npm\node_modules\jscpd\node_modules\@jscpd\tokenizer\dist\tokenize.js:65:36)
    at tokens.filter.map (C:\Users\ehammes\AppData\Roaming\npm\node_modules\jscpd\node_modules\@jscpd\tokenizer\dist\tokenize.js:111:32)
    at Array.map (<anonymous>)
    at tokenize (C:\Users\ehammes\AppData\Roaming\npm\node_modules\jscpd\node_modules\@jscpd\tokenizer\dist\tokenize.js:111:10)
    at Object.createTokenMapBasedOnCode (C:\Users\ehammes\AppData\Roaming\npm\node_modules\jscpd\node_modules\@jscpd\tokenizer\dist\tokenize.js:116:20)
    at Tokenizer.generateMaps (C:\Users\ehammes\AppData\Roaming\npm\node_modules\jscpd\node_modules\@jscpd\tokenizer\dist\index.js:12:27)
    at Detector.<anonymous> (C:\Users\ehammes\AppData\Roaming\npm\node_modules\jscpd\node_modules\@jscpd\core\dist\detector.js:33:46)
    at Generator.next (<anonymous>)
    at C:\Users\ehammes\AppData\Roaming\npm\node_modules\jscpd\node_modules\@jscpd\core\dist\detector.js:8:71
    at new Promise (<anonymous>)

```
When I run `npx jscpd --gitignore ./` in git bash

**Desktop:**
 - OS: Windows 10 Enterprise
 - NodeJS Version 10.17.0
 - jscpd version 3.3.19

I'm not sure how to narrow down why this problem is occurring within my specific project.

**Comment by kucherenko** [comment]
Thank you for the report, will fix it

**Comment by kucherenko** [comment]
fixed in jscpd@3.3.20
